.github/workflows/commitlint.yml

```diff
@@ -0,0 +1,16 @@
+name: Commitlint
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: wagoid/commitlint-github-action@v1.6.0
```

Comment:
Where did you find the docs for this? Please add that in the commit message or a comment in this file. I wonder what repository `wagoid` is...